onnxruntime/test/providers/provider_test_utils.h

```diff
@@ -148,8 +148,10 @@ const VectorOfMapTypeProto<TKey, TVal> s_vec_map_type_proto;
 // See current usage for an example, should be self explanatory
 class OpTester {
  public:
-  OpTester(const char* op, int opset_version = 7, const char* domain = onnxruntime::kOnnxDomain)
+  explicit OpTester(const char* op, int opset_version = 7, const char* domain = onnxruntime::kOnnxDomain)
       : op_(op), domain_(domain), opset_version_(opset_version) {}
+  OpTester(OpTester&& other) = default;
+  OpTester& operator=(OpTester&& other) = default;
   ~OpTester();
```

Comment:
It seems pretty heavy to add this if it's just to support calling MakeOpTester given that contains one line. 

OpTester MakeOpTester() {
  return OpTester{"Where", 9};
}

Reply:
Heavy in what sense? Is there a reason that OpTester should not be movable?

Regarding the one line function, I thought it would be less error prone than having to type OpTester{"Where", 9} each time.

Reply:
Heavy in that any future changes to OpTester now might need to consider whether they handle a move correctly. Typing 'OpTester{"Where", 9} correctly each time (or most likely cut-and-pasting) within one unit test file seems simpler to me on balance, given if you get it wrong the unit test (and not some production code) will break immediately.

Another option is to simply derive a local class to plug in the ctor param values and use that in the Where unit tests. 

class WhereOpTester : public OpTester { public: WhereOpTester() : OpTester{"Where", 9} {}};